src/main/resources/templates/home.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <link rel="stylesheet" th:href="@{/webjars/bootstrap/3.3.7-1/css/bootstrap.min.css}"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>SaladBoard</title>
+</head>
+<body>
+This is home page.
+
+<hr>
+
+<tr th:each="post:${allPosts}" th:object="${post}">
+    <td>body :</td>
+    <td th:text="*{body}"></td>
+    <br>
+    <td>user_id :</td>
+    <td th:text="*{userId}"></td>
+    <br>
+    <td>posted_at :</td>
+    <td th:text="*{postedAt}"></td>
+    <br>
+    <hr>
+</tr>
```

Comment:
使っちゃいけないことは無いですが、divの方がCSSで調整しやすいです
表示調整は諦めるとのことでしたので、一つ簡単な回答例を示すなら
```suggestion
<div style="display: flex;flex-wrap: wrap" th:each="post:${allPosts}" th:object="${post}">
    <div>user_id :<span th:text="*{userId}"></span></div>
    <div style="margin-left: auto">posted_at :<span th:text="*{postedAt}"></span></div>
    <div style="width: 100%">body :<span th:text="*{body}"></span></div>
    <hr style="width: 100%">
</div>
```
軽いHTML/CSSの知識はサーバーサイドエンジニアでも使うことあるので、調べてみてください